reference/tools/data-migration/features/online-ddl-scheme.md

```diff
@@ -91,27 +91,27 @@ DM 在同步过程中会把上述 table 分成 3 类：
 
 ### DM 对于 **online-ddl-scheme: gh-ost** 的处理
 
-1. 不执行 `_test4_ghc` 的创建操作。
-2. 不执行 `_test4_gho` 的创建操作，根据 ghost_schema、ghost_table 以及 dm_worker 的 server_id 删除下游 dm_meta.{task_name}\_onlineddl 的记录，清理内存中的相关信息。
+- 不执行 `_test4_ghc` 的创建操作。
```

Comment:
I think the old format is easier for reader to understand. Because The steps match for the SQLs before. Not the points to list up.

Reply:
Thanks, already restored the ordered list.